generators/csharp/sdk/src/reference/getReferenceConfigBuilder.ts

```diff
@@ -0,0 +1,128 @@
+import { ReferenceConfigBuilder } from "@fern-api/generator-commons";
+import { FernGeneratorCli } from "@fern-fern/generator-cli-sdk";
+import { HttpEndpoint, HttpService, ServiceId } from "@fern-fern/ir-sdk/api";
+import { EndpointSignatureInfo } from "../endpoint/EndpointSignatureInfo";
+import { SingleEndpointSnippet } from "../endpoint/snippets/EndpointSnippetsGenerator";
+import { SdkGeneratorContext } from "../SdkGeneratorContext";
+
+export function getReferenceConfigBuilder({ context }: { context: SdkGeneratorContext }): ReferenceConfigBuilder {
```

Comment:
nit should we call this method `buildReference()`

Reply:
Yea good call, originally felt that it would read weird since we're actually just returning the builder, but `buildReference` is still a lot more clear than `getReferenceConfigBuilder`. I'll update.